roles/georchestra/tasks/wars.yml

```diff
@@ -61,7 +61,7 @@
 - name: symlink webapps in each tomcat instance
   file:
     owner: tomcat
-    src: "/usr/share/lib/{{ item.value.pkg }}/{{ item.key }}-generic.war"
+    src: "/usr/share/lib/{{ item.value.pkg }}/{{ item.key }}{% if item.key != 'import' and item.key != 'geowebcache' %}-generic{% endif %}.war"
```

Comment:
too bad, it's not harmonized across packages :-/

Reply:
same thing for geonetwork actually, cf f6b81af